tests/foreman/api/test_template_combination.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""Tests for template combination
+
+@Requirement: TemplateCombination
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: API
+
+@TestType: Functional
+
+@CaseImportance: Medium
+
+@Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.decorators import tier1, skip_if_bug_open
+from robottelo.test import APITestCase
+
+
+def _delete_entities(*entities):
+    for ent in entities:
+        ent.delete()
+
+
+class TestTemplateCombination(APITestCase):
+    """TemplateCombination tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestTemplateCombination, cls).setUpClass()
+        cls.hostgroup = entities.HostGroup().create()
+        cls.env = entities.Environment().create()
+
+    @classmethod
+    def tearDownClass(cls):
+        super(TestTemplateCombination, cls).tearDownClass()
+        _delete_entities(cls.hostgroup, cls.env)
+
+    def setUp(self):
```

Comment:
setUp is doing what it must to: creating the all entities required to test. In case someone add more tests here, no need to copy and paste entities creation logic. Plus, all entities are cleaned up after test is finished. Delete procedure itself is already tested on nailgun. Just added read test to check returned fields.